diff --git src/.vuepress/theme.ts src/.vuepress/theme.ts
index 2c4c294..4c706d1 100644
--- src/.vuepress/theme.ts
+++ src/.vuepress/theme.ts
@@ -4,7 +4,7 @@ import navbar from "./navbar.js";
 import sidebar from "./sidebar.js";
 
 export default hopeTheme({
-    hostname: "https://xddcloud.net",
+    hostname: "https://venti.ink",
 
     author: {
         name: "小电动",
@@ -95,15 +95,6 @@ export default hopeTheme({
     // 在这里配置主题提供的插件
     plugins: {
         blog: true,
-        
-        comment: {
-            provider: "Giscus",
-            repo: "handsomexdd1024/battery",
-            repoId: "R_kgDONRqLpQ",
-            category: "General",
-            categoryId: "DIC_kwDONRqLpc4CkeRi",
-            strict: false,
-        },
 
         components: {
             components: ["Badge", "VPCard"],
